thoth-api/migrations/0.2.14/up.sql

```diff
@@ -9,3 +9,25 @@ CREATE TABLE publisher_account (
 SELECT diesel_manage_updated_at('publisher_account');
 
 ALTER TABLE account RENAME COLUMN is_admin TO is_superuser;
+
+ALTER TABLE contribution
+    ADD COLUMN first_name TEXT,
+    ADD COLUMN last_name TEXT,
+    ADD COLUMN full_name TEXT;
+
+UPDATE contribution
+    SET first_name = contributor.first_name,
+        last_name = contributor.last_name,
+        full_name = contributor.full_name
+    FROM contributor
+    WHERE contribution.contributor_id = contributor.contributor_id;
+
+ALTER TABLE contribution
+    ALTER COLUMN last_name SET NOT NULL,
+    ALTER COLUMN full_name SET NOT NULL,
+    ADD CONSTRAINT name_length_check
```

Comment:
I'm inclined to have this as three separate constraints rather than a single one, just to be consistent with the existing checks - what do you think?

Reply:
Agreed - there's a slight difference in behaviour which I hadn't noticed, and consistency is important. I'll change this now.